workshop/Lab2/README.md

```diff
@@ -76,30 +75,32 @@ to your deployment, as shown in the following diagram:
 
 ![HA with more clusters and regions](../images/cluster_ha_roadmap.png)
 
-# 2. Update and roll back apps
+## 2. Update and roll back apps
 
-Kubernetes allows you to do rolling upgrade of your application to a new
-container image. This allows you to easily update the running image and also allows you to
-easily undo a rollout if a problem is discovered during or after deployment.
+Kubernetes allows you to conduct a rolling upgrade of your application to a new
+container image. This allows you to easily update the running image and
+undo a rollout if you discover a problem during or after deployment.
 
-In the previous lab, we used an image with a `v1` tag. For our upgrade
+In the previous lab, we used an image with a `v1` tag. For our upgrade,
 we'll use the image with the `v2` tag.
 
-To update and roll back:
-1. Using `kubectl`, you can now update your deployment to use the
-   `v2` image. `kubectl` allows you to change details about existing
+Follow these steps to update and roll back your app:
+
+1. Update your deployment to use the
+   `v2` image. Use `kubectl`, which allows you to change details about existing
    resources with the `set` subcommand. We can use it to change the
    image being used.
 
     ```$ kubectl set image deployment/guestbook guestbook=ibmcom/guestbook:v2```
 
-   Note that a pod could have multiple containers, each with its own name.
+   Note that a pod can have multiple containers, each with its own name.
    Each image can be changed individually or all at once by referring to the name.
-   In the case of our `guestbook` Deployment, the container name is also `guestbook`.
-   Multiple containers can be updated at the same time.
-   ([More information](https://kubernetes.io/docs/user-guide/kubectl/kubectl_set_image/).)
+   In the case of our `guestbook` deployment, the container name is also `guestbook`.
+   You can also update multiple containers simultaneously. Visit the [Kubernetes guide
+   on how to update multiple containers]().
```

Comment:
Missing the URL, no? Actually, is the next line what you really wanted and just forgot to delete this sentence?